src/libs/actions/App.ts

```diff
@@ -424,7 +424,7 @@ function createWorkspaceWithPolicyDraftAndNavigateToIt(
  * @param [file] Optional, avatar file for workspace
  */
 function savePolicyDraftByNewWorkspace(policyID?: string, policyName?: string, policyOwnerEmail = '', makeMeAdmin = false, currency = '', file?: File) {
-    createWorkspace(policyOwnerEmail, makeMeAdmin, policyName, policyID, '', currency, file);
+    createWorkspace(policyOwnerEmail, makeMeAdmin, policyName, policyID, CONST.ONBOARDING_CHOICES.MANAGE_TEAM, currency, file);
```

Comment:
@ishpaul777 @francoisl do you see a need for adding a comment on why we're passing MANAGE_TEAM ?

Reply:
No strong feelings either, though I think it might be more beneficial if we add an explanation in the doc of the function definition, in `Policy.ts`. Right now it says `@param [engagementChoice] Purpose of using application selected by user in guided setup flow` - we can add a note saying that the default is `MANAGE_TEAM` and you can change for specific policy creation flows